docs/sources/reference/components/prometheus.exporter.snowflake.md

```diff
@@ -20,18 +22,34 @@ prometheus.exporter.snowflake "LABEL" {
 }
 ```
 
+### RSA Authentication
+
+```alloy
+prometheus.exporter.snowflake "LABEL" {
+    account_name =         ACCOUNT_NAME
+    username =             USERNAME
+    private_key_path =     /PATH/TO/rsa_key.p8
+    private_key_password = PASSWORD
+    warehouse =            WAREHOUSE
+}
+```
+
```

Comment:
```suggestion

Replace the following:

* _`<ACCOUNT_NAME>`_: The Snowflake account name.
* _`<USERNAME>`_: The username used to query metrics.
* _`<PRIVATE_KEY_PATH>`_: The path to the RSA private key..
* _`<PASSWORD>`_: The password for the user used to query metrics.
* _`<WAREHOUSE>`_: The virtual warehouse to use when querying metrics.

```

This is the second part of the style update. This style should be cascaded to at least the other new content (Password Authentication section) in this topic.  The existing content (the example near the bottom of the topic) can be left for a future update, or if there's time, included in this PR.

Reply:
Lines 127-129 could use a tweak to add underscores and < >.  I can't comment those lines in this PR. Otherwise looks OK.